eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -246,12 +243,12 @@ stages:
                       echo "##vso[task.setvariable variable=NpmToken]$npmToken"
                       echo "##vso[task.setvariable variable=Registry]$registry"
                     displayName: Detecting package archive_${{artifact.name}}
-                  - task: PowerShell@2
-                    displayName: Publish_${{artifact.name}} to dev feed
-                    inputs:
-                      targetType: filePath
-                      filePath: eng/tools/publish-to-npm.ps1
-                      arguments: -pathToArtifacts $(Package.Archive) -accessLevel "public" -tag "dev" -registry "$(Registry)" -npmToken "$(NpmToken)"
+                  - template: /eng/pipelines/templates/steps/npm-release-task.yml
```

Comment:
It seems like we are still doing this work once per package, see each loop above over parameters.Artifacts. Shouldn't we do this as a set now?

Reply:
yes we're still doing this once per package because there's still steps before npm-release-task.yml (including verify changelog, verify rest api spec location, and create tags and git release) which works in the loop context. Unless we want to change the entire archetype-js-release and the other related yml specs?